src/tests/test_crypto.py

```diff
@@ -62,6 +64,32 @@ def _hashdigest(data):
 class TestHighlevelcrypto(unittest.TestCase):
     """Test highlevelcrypto public functions"""
 
+    # def test_sign(self):
+    #     """Check the signature of the sample_msg created with sample key"""
+    #     self.assertEqual(
+    #         highlevelcrypto.sign(sample_msg, sample_privsigningkey), sample_sig)
+
+    def test_double_sha512(self):
+        """Reproduce the example on page 1 of the Specification"""
+        self.assertEqual(
+            highlevelcrypto.double_sha512(b'hello'), sample_double_sha512)
```

Comment:
ideally `b'hello'` should also be a variable in samples

Reply:
It's from here: https://pybitmessage-test.readthedocs.io/en/doc/protocol.html#hashes